### PR TITLE
feat: Add QRadar exporter

### PR DESCRIPTION
**Pull request by shazlehu**
### Proposed Change

Adds QRadar exporter.

Test by using the following custom destination:

```yaml
qradar:
    raw_log_field: body
    retry_on_failure:
        enabled: false
    sending_queue:
        enabled: false
    syslog:
        endpoint: qr75-appliance.bluemedora.localnet:514
        transport: udp
```

Send a JSON payload from Telemetry generator.

- Log into QRadar at http://qr75-appliance.bluemedora.localnet
- Select "Admin" from the Tabs
- Select "Log Sources" from the "Data Sources" section
- Select "Events" from the "..." menu on the "SIM Generic Log DSM-7 :: qr75-appliance" line
- Logs should appear there, double-clicking a line should show the correct payload

##### Checklist
- [ ] Changes are tested
- [ ] CI has passed

### CODE REVIEW

File: exporter/qradar/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# QRadaar Exporter
+
+The QRadar Exporter is designed for forwarding logs to a QRadar endpoint using either Syslog or File-based methods. This exporter supports customization of data export types and various configuration options to tailor the connection and data handling to specific needs.
+
+## Minimum Agent Versions
+
+- Introduced: [v1.42.0](https://github.com/observIQ/bindplane-agent/releases/tag/v1.42.0)
```
Comment (BinaryFissionGames) [inline exporter/qradar/README.md:7]: Should be v1.61

File: exporter/qradar/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# QRadaar Exporter
+
+The QRadar Exporter is designed for forwarding logs to a QRadar endpoint using either Syslog or File-based methods. This exporter supports customization of data export types and various configuration options to tailor the connection and data handling to specific needs.
```
Comment (BinaryFissionGames) [inline exporter/qradar/README.md:3]: How does the file-based method work? Is it applicable to qradar?

**Review by BinaryFissionGames — CHANGES_REQUESTED** [review]

File: exporter/qradar/README.md (hunk 2 above)
Comment (shazlehu) [reply exporter/qradar/README.md]: Removed File-based method

**Review by BinaryFissionGames — APPROVED** [review]
Working well for me 👍

**Pull request merged**